net-misc/xmrig/xmrig-20251105.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="CPU and GPU miner"
+DESCRIPTION="RandomX, KawPow, CryptoNight and GhostRider unified CPU/GPU miner and RandomX benchmark"
+HOMEPAGE="https://github.com/xmrig/xmrig"
+COPYRIGHT="2010 Jeff Garzik
+	2012-2014 pooler
+	2014 Lucas Jones
+	2014-2016 Wolf9466
+	2016 Jay D Dee
+	2017-2018 XMR-Stak
+	2018 Lee Clagett
+	2018-2024 SChernykh
+	2016-2024 XMRig"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="4cdc35f96648f8363d26526d1bfa90502833bac9"
+SOURCE_URI="https://github.com/xmrig/xmrig/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="b80a166521c7d447775e5c012b36ef3898388d8f8f72bf4e5fbe9769de94a60a"
+SOURCE_DIR="xmrig-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	xmrig$secondaryArchSuffix = $portVersion
+	cmd:xmrig = $portVersion
+"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libuv$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libuv$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+"
+
+BUILD()
+{
+	mkdir -p build
+	cmake -B build \
+		-DWITH_HWLOC=OFF\
+		-DCMAKE_BUILD_TYPE=Release
+
+	make -C build $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
+	install -Dm755 build/xmrig $binDir/xmrig
```

Comment:
```suggestion
	mkdir -p $prefix/bin
	install -Dm755 build/xmrig $prefix/bin/xmrig
```
`$binDir` contains the secondaryArch subdirectory on x86, which is not needed in this case.